addons/web/static/tests/helpers/utils.js

```diff
@@ -483,6 +483,17 @@ export function editSelect(el, selector, value) {
     return triggerEvent(select, null, "change");
 }
 
+export async function editSelectComponent(el, selector, value) {
+    const dropdown = el.querySelector(selector);
+    await click(dropdown.querySelector(".dropdown-toggle"));
+    await nextTick();
```

Comment:
should not be necessary anymore

Reply:
It is still useful for the AdvancedSelect component, because some tests need to interact with it

Reply:
I meant "the nextTick call is not necessary anymore since you would use the onBeforeOpened dropdown prop"

Reply:
Oh okay :)